smart-send-logistics/includes/frontend/class-ss-shipping-endpoint.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+if (!defined('ABSPATH')) {
+    exit; // Exit if accessed directly
+}
+include '../../pickup-point-block/smart-send-utility-session.php';
+/**
+ * Class SS_Shipping_Api_Endpoint
+ *
+ * Handles the REST API endpoints for Smart Send Logistics.
+ *
+ * This class is responsible for registering the REST API endpoints and handling
+ * requests to fetch the closest pickup points based on provided address parameters.
+ *
+ * @package    SmartSendLogistics
+ * @subpackage SS_Shipping_Endpoint
+ * @category   REST API
+ * @author     Smart Send
+ */
+class SS_Shipping_Api_Endpoint
+{
+
+    /**
+     * @var FrontendInterface $frontend An instance of the frontend class handling agent fetching and session management.
```

Comment:
Is this variable not a `SS_Shipping_Frontend` instance?

Reply:
This is variable which is used to store the instance of the  `SS_Shipping_Frontend`  Class.

Reply:
I get that. But then why have you docblocked that it is an instance of `FrontendInterface`?

Reply:
This is still incorrect. It should say `@var SS_Shipping_Frontend $frontend` instead of `@var FrontendInterface $frontend` right?